geoadmin/src/main/java/org/activityinfo/geoadmin/model/ActivityInfoClient.java

```diff
@@ -1,14 +1,33 @@
 package org.activityinfo.geoadmin.model;
 
 import com.bedatadriven.geojson.GeoJsonModule;
+import com.google.common.base.Preconditions;
 import com.google.common.collect.Lists;
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
 import com.sun.jersey.api.client.Client;
 import com.sun.jersey.api.client.GenericType;
+import com.sun.jersey.api.client.WebResource;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of Sun classes by ones from the Java API. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1191)